### ISSUE TITLE
Cannot load a namespace with :preload in :node-library

### ISSUE THREAD

**Issue by arichiardi** [thheller/shadow-cljs#302]
Hi Thomas, I am not sure what is going on but it seems that I cannot `:preloads` namespace with target `:node-library` (same for `:node-script, I tried).

This is what I have:

```shell
 $ find src -name '*preload*'
src/ep_cloud/preload.cljs
```

```clojure
...
:devtools {:preloads [ep-cloud.preload]}
...
```

```shell
$ find .shadow-cljs/ -name '*preload*'
-nothing-
$ find init-store/ -name '*preload*'
-nothing-
```

Am I doing something wrong?

**Comment by thheller** [comment]
Fixed in `2.4.1`. Forgot that only `:browser` supported `:preloads` previously. Now both `:node-script` and `:node-library` do.

**Comment by arichiardi** [comment]
Thank you!
